test/python/test_subarchitectures.py

```diff
@@ -0,0 +1,94 @@
+import retworkx as rx
+from mqt.qmap.subarchitectures import SubarchitectureOrder
+
+
+def test_singleton_graph():
+    """Verify that singleton graph has trivial ordering."""
+    g = rx.PyGraph()
+    g.add_node(0)
```

Comment:
Although it is very little code duplication, I believe it would make sense to extract any graph that is being used multiple times into a pytest fixture. See https://docs.pytest.org/en/latest/explanation/fixtures.html

Reply:
Thanks for the hint.